Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -1,66 +1,102 @@
-    assert len(alert_ids) == len(alerts_details)
-
-
-def test_alerts_in_sentinel():
-    alert_ids = get_alerts(api_key)
-    for alert_id in alert_ids:
-        assert search_alert_id_in_incident(alert_id, resource_group, workspace_name) is not None, f"alert_id --> {alert_id} doesn't exist in sentinel."
-
-bash_command = "./deploy_playbooks.sh"
-process = subprocess.Popen(bash_command.split(), stdout=subprocess.PIPE)
-output, error = process.communicate()
-
-if output:
-    print("output --> %s" % output.decode())
-
-if error:
-    print("error --> %s" % error.decode())
-
-f = open('../cohesity.json',)
-data = json.load(f)
-resource_group = data['resource_group']
-workspace_name = data['workspace_name']
-api_key = data['apiKey']
-f.close()
-
-assert has_dup_incidents(resource_group, workspace_name) == False
-
-test_cohesity_close_helios_incident()
-test_all_incidents_in_helios()
-test_alerts_in_sentinel()
+import subprocess
+import time
+import unittest
+
+
+class TestCohesity(unittest.TestCase):
+    def setUp(self):
+        """
+        This method is called before each test case and is used to set up any necessary resources or configurations.
+        In this case, the configuration for the Cohesity system is loaded from a JSON file, and the necessary playbooks
+        are deployed.
+        """
+        # Load config from JSON file
+        with open('../cohesity.json') as f:
+            config = json.load(f)
+            self.resource_group = config['resource_group']
+            self.workspace_name = config['workspace_name']
+            self.api_key = config['api_key']
+
+        # Verify that config values are not empty
+        self.assertNotEqual(self.resource_group, "", "resource_group is empty")
+        self.assertNotEqual(self.workspace_name, "", "workspace_name is empty")
+        self.assertNotEqual(self.api_key, "", "api_key is empty")
+
+        # Deploy playbooks
+        bash_command = "./deploy_playbooks.sh"
+        process = subprocess.Popen(bash_command.split(), stdout=subprocess.PIPE)
+        output, error = process.communicate()
+
+        if output:
+            print("output --> %s" % output.decode())
+
+        if error:
+            print("error --> %s" % error.decode())
+
+    def test_cohesity_close_helios_incident(self):
+        """
+        This test case verifies that the 'Cohesity_Close_Helios_Incident' playbook can successfully close a Helios
+        incident. It first retrieves an incident ID and verifies that the corresponding Helios alert is in the 'kOpen'
+        state. It then runs the playbook and waits for 30 seconds before verifying that the Helios alert is in the
+        'kSuppressed' state.
+        """
+        # self.skipTest("Skipping test_cohesity_close_helios_incident")
+        playbook_name = "Cohesity_Close_Helios_Incident"
+        subscription_id = get_subscription_id()
+        vid, alert_id = get_one_incident_id(self.resource_group, self.workspace_name)
```

Comment:
Could you please use a more expressive name than vid? I'm not sure what vid means here.

Reply:
done